src/pages/nft/class/_classId/index.vue

```diff
@@ -1239,6 +1298,23 @@ export default {
         }
       }, 100);
     },
+    handleSubmitTipping(price) {
+      this.customPrice = Number(price);
+      this.handleCollectFromEdition();
+    },
+    handleSkipTipping() {
```

Comment:
Do we need to reset `customPrice` here?

Reply:
[🎨 Reset custom when skipping](https://github.com/likecoin/liker-land/pull/1554/commits/2da3ea41b6062db8e824a4c8878d17088188fd5d)